### PR TITLE
app-i18n/fbterm: version bump

### PR DESCRIPTION
**Pull request by DarthGandalf**
Change upstream to something slightly more maintained. An example
improvement: a correct TERM value inside the terminal ("fbterm" instead
of "linux")

Package-Manager: Portage-2.3.89, Repoman-2.3.20
Signed-off-by: Alexey Sokolov <sokolov@google.com>

### CODE REVIEW

File: app-i18n/fbterm/fbterm-1.7_p20190503.ebuild [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit autotools fcaps toolchain-funcs
+
+DESCRIPTION="Fast terminal emulator for the Linux framebuffer"
+HOMEPAGE="https://github.com/gjedeer/fbterm"
+SRC_URI="https://github.com/gjedeer/fbterm/archive/ccea326dd73f4d6b6442fde7ba7c2be9cd35c6df.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/fbterm-ccea326dd73f4d6b6442fde7ba7c2be9cd35c6df"
```
Comment (sbraz) [inline app-i18n/fbterm/fbterm-1.7_p20190503.ebuild:11]: Maybe you'll want to use a `MY_COMMIT` variable to make future bumps easier.
Kinda like what I did here:
https://github.com/gentoo/gentoo/blob/4f620a3ba9edc210e2a55eff7c5c71efd713f4ef/net-analyzer/nagios-plugin-check_lm_sensors/nagios-plugin-check_lm_sensors-4.1.1_p20191028.ebuild#L9

File: app-i18n/fbterm/fbterm-1.7_p20190503.ebuild [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit autotools fcaps toolchain-funcs
+
+DESCRIPTION="Fast terminal emulator for the Linux framebuffer"
+HOMEPAGE="https://github.com/gjedeer/fbterm"
```
Comment (sbraz) [inline app-i18n/fbterm/fbterm-1.7_p20190503.ebuild:9]: If you're updating the URL, you should do it in all ebuilds. I'm not sure which HOMEPAGE packages.gentoo.org and tools like eix use when it has different values (probably that of the first ebuild they find).

**Comment by DarthGandalf** [comment]
Thanks, both done.

File: app-i18n/fbterm/fbterm-1.7_p20190503.ebuild (hunk 2 above)
Comment (Arfrever) [reply app-i18n/fbterm/fbterm-1.7_p20190503.ebuild:9]: This is misadvice.
HOMEPAGE in older versions should point to this upstream from whom those versions come from.
Tools show HOMEPAGE from newest ebuild.

Comment (DarthGandalf) [reply app-i18n/fbterm/fbterm-1.7_p20190503.ebuild:9]: Good to know. Removed that commit.

**Comment by DarthGandalf** [comment]
@hattya hi, could you take a look?

File: app-i18n/fbterm/fbterm-1.7_p20190503.ebuild [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+MY_COMMIT="ccea326dd73f4d6b6442fde7ba7c2be9cd35c6df"
+SRC_URI="https://github.com/gjedeer/fbterm/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/fbterm-${MY_COMMIT}"
```
Comment (hattya) [inline app-i18n/fbterm/fbterm-1.7_p20190503.ebuild:13]: You can use vcs-snapshot eclass.

I'd prefer to declare `MY_COMMIT` between `inherit` and `DESCRIPTION`, and as `EGIT_COMMIT` for this.

```suggestion
inherit autotools fcaps toolchain-funcs vcs-snapshot

EGIT_COMMIT="ccea326dd73f4d6b6442fde7ba7c2be9cd35c6df"

DESCRIPTION="Fast terminal emulator for the Linux framebuffer"
HOMEPAGE="https://github.com/gjedeer/fbterm"
SRC_URI="https://github.com/gjedeer/${PN}/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
```

File: app-i18n/fbterm/fbterm-1.7_p20190503.ebuild [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+DEPEND="
+	media-libs/fontconfig
+	media-libs/freetype:2
+	>=sys-libs/ncurses-6.1
+	gpm? ( sys-libs/gpm )
+	video_cards_vesa? ( dev-libs/libx86 )
+"
+BDEPEND="
+	virtual/pkgconfig
+"
```
Comment (hattya) [inline app-i18n/fbterm/fbterm-1.7_p20190503.ebuild:29]: I'd prefer no newline after or before double quote.

```suggestion
DEPEND="media-libs/fontconfig
	media-libs/freetype:2
	>=sys-libs/ncurses-6.1
	gpm? ( sys-libs/gpm )
	video_cards_vesa? ( dev-libs/libx86 )"
BDEPEND="virtual/pkgconfig"
```

File: app-i18n/fbterm/fbterm-1.7_p20190503.ebuild (hunk 3 above)
Comment (DarthGandalf) [reply app-i18n/fbterm/fbterm-1.7_p20190503.ebuild]: Done. This is not git-r3 though, why EGIT_COMMIT?

File: app-i18n/fbterm/fbterm-1.7_p20190503.ebuild (hunk 4 above)
Comment (DarthGandalf) [reply app-i18n/fbterm/fbterm-1.7_p20190503.ebuild]: Done.

File: app-i18n/fbterm/fbterm-1.7_p20190503.ebuild (hunk 3 above)
Comment (hattya) [reply app-i18n/fbterm/fbterm-1.7_p20190503.ebuild]: There is no particular reason.

File: app-i18n/fbterm/fbterm-1.7_p20190503.ebuild (hunk 4 above)
Comment (Arfrever) [reply app-i18n/fbterm/fbterm-1.7_p20190503.ebuild]: Ebuild is missing:
```
RDEPEND="${DEPEND}"
```

**Pull request closed without merging**